docs/tutorials/python_userguide/coherence.txt

```diff
@@ -1,2 +1,71 @@
-6. Tokens Coocurancy and Coherence Coumputation
-=======
+6. Tokens Coocurancy and Coherence Computation
+==============================================
+
+* **Interpretability and Coherence of Topics**
+
+The one of main requirements of topic models is interpretability (i.e., do the topics contain tokens that, according to subjective human judgment, are representative of single coherent concept). `Newman at al. <http://www.aclweb.org/anthology/N10-1012>`_ showed the human evaluation of interpretability is well correlated with next automated quality measure called coherence. The coherence formula for topic is defined as
+
+:math:`\mathcal{C}_t = \cfrac{2}{k(k - 1)} \sum_{i = 1}^{k - 1} \sum_{j = 1}^{k} \mathrm{value}(w_i, w_j)`,
+
+where value is some pairwise information about tokens in collection dictionary, which is provided by user according to his goals.
+
+* **Tokens Coocurancy Dictionary**
+
+BigARTM provides ability of automatic coherence computation. Only one thing you should prepare is file with coocurancies of tokens. Lines of that file should have the next format: ::
+
+	id1 id2 value
+
+where ``id1`` and ``id2`` are indexes of tokens in vocabulary file in UCI format (but indexing starts from 0!). Details about UCI format can be found here: :doc:`../datasets`. ``value`` is some pairwise information about tokens with indexes ``id1`` and ``id2``. For instance, ``value`` can be computed by any corpora of texts by formulas:
```

Comment:
probably we should say *(with 0-based indexing!)* instead of *(but indexing starts from 0!)*